src/screens/OrgPost/OrgPost.tsx

```diff
@@ -382,116 +397,54 @@ function orgPost(): JSX.Element {
-                  onChange={async (
-                    e: React.ChangeEvent<HTMLInputElement>
-                  ): Promise<void> => {
-                    setPostFormState((prevPostFormState) => ({
-                      ...prevPostFormState,
-                      postImage: '',
-                    }));
-                    const file = e.target.files?.[0];
-                    if (file) {
-                      setPostFormState({
-                        ...postformState,
-                        postImage: await convertToBase64(file),
-                      });
-                    }
-                  }}
-                />
+          </Modal.Body>
+          <Modal.Body>
+            <Form.Label htmlFor="addMedia">{t('addMedia')}</Form.Label>
+            <Form.Control
+              id="addMedia"
+              name="addMedia"
+              type="file"
+              accept="image/*,video/*"
+              placeholder={t('addMedia')}
+              multiple={false}
+              onChange={handleAddMediaChange}
+              data-testid="addMediaField"
+            />
 
-                {postformState.postImage && (
-                  <div className={styles.preview} data-testid="org">
-                    <img
-                      src={postformState.postImage}
-                      alt="Post Image Preview"
-                    />
-                    <button
-                      className={styles.closeButton}
-                      onClick={(): void => {
-                        setPostFormState({
-                          ...postformState,
-                          postImage: '',
-                        });
-                        const fileInput = document.getElementById(
-                          'postphoto'
-                        ) as HTMLInputElement;
-                        if (fileInput) {
-                          fileInput.value = '';
-                        }
-                      }}
-                      data-testid="closePreview"
-                    >
-                      <i className="fa fa-times"></i>
-                    </button>
-                  </div>
+            {postformState.addMedia && file && (
+              <div className={styles.preview} data-testid="mediaPreview">
+                {/* Display preview for both image and video */}
+                {file.type.startsWith('image') ? (
+                  <img
+                    src={postformState.addMedia}
+                    data-testid="imagePreview"
+                    alt="Post Image Preview"
+                  />
+                ) : (
+                  <video controls data-testid="videoPreview">
+                    <source src={postformState.addMedia} type={file.type} />(
+                    {t('tag')})
+                  </video>
                 )}
-              </>
-            )}
-            {!postformState.postImage && (
-              <>
-                <Form.Label htmlFor="postvideo">{t('video')}</Form.Label>
```

Comment:
I think you can delete `video` from `en.json` and other files too. If it's not in use anywhere else in this component/screen.